nlp_apps.ipynb

```diff
@@ -531,7 +559,7 @@
        "(4, 16, 52)"
       ]
      },
-     "execution_count": 6,
+     "execution_count": 41,
```

Comment:
A slightly picky complaint, but you can rerun a notebook to serialize the execution counts.